src/pcapi/core/offerers/repository.py

```diff
@@ -45,66 +50,120 @@ def get_filtered_venues(
-            query = query.filter(UserOfferer.validationToken.is_(None))
+            query = query.filter(offerers_models.UserOfferer.validationToken.is_(None))
         else:
-            query = query.filter(UserOfferer.validationToken.isnot(None))
+            query = query.filter(offerers_models.UserOfferer.validationToken.isnot(None))
 
     if active_offerers_only:
-        query = query.filter(Offerer.isActive.is_(True))
+        query = query.filter(offerers_models.Offerer.isActive.is_(True))
 
     if offerer_id:
-        query = query.filter(Venue.managingOffererId == offerer_id)
+        query = query.filter(offerers_models.Venue.managingOffererId == offerer_id)
 
-    return query.order_by(Venue.name).all()
+    return query.order_by(offerers_models.Venue.name).all()
 
 
 def get_api_key_prefixes(offerer_id: int) -> list[str]:
     return [
         prefix or value[:8]
-        for prefix, value in ApiKey.query.filter_by(offererId=offerer_id)
-        .with_entities(ApiKey.prefix, ApiKey.value)
+        for prefix, value in offerers_models.ApiKey.query.filter_by(offererId=offerer_id)
+        .with_entities(offerers_models.ApiKey.prefix, offerers_models.ApiKey.value)
         .all()
         if prefix or value
     ]
 
 
-def find_offerer_by_siren(siren: str) -> Optional[Offerer]:
-    return Offerer.query.filter_by(siren=siren).one_or_none()
+def find_offerer_by_siren(siren: str) -> Optional[offerers_models.Offerer]:
+    return offerers_models.Offerer.query.filter_by(siren=siren).one_or_none()
+
+
+def find_user_offerer_by_validation_token(token: str) -> Optional[offerers_models.UserOfferer]:
+    return offerers_models.UserOfferer.query.filter_by(validationToken=token).one_or_none()
+
+
+def find_offerer_by_validation_token(token: str) -> Optional[offerers_models.UserOfferer]:
+    return offerers_models.Offerer.query.filter_by(validationToken=token).one_or_none()
+
+
+def find_venue_by_id(venue_id: int) -> Optional[offerers_models.Venue]:
+    return offerers_models.Venue.query.filter_by(id=venue_id).one_or_none()
+
+
+def find_venue_by_siret(siret: str) -> Optional[offerers_models.Venue]:
+    return offerers_models.Venue.query.filter_by(siret=siret).one_or_none()
+
 
+def find_venue_by_managing_offerer_id(offerer_id: int) -> Optional[offerers_models.Venue]:
+    return offerers_models.Venue.query.filter_by(managingOffererId=offerer_id).first()
 
-def find_user_offerer_by_validation_token(token: str) -> Optional[UserOfferer]:
-    return UserOfferer.query.filter_by(validationToken=token).one_or_none()
 
+def find_virtual_venue_by_offerer_id(offerer_id: int) -> Optional[offerers_models.Venue]:
+    return offerers_models.Venue.query.filter_by(managingOffererId=offerer_id, isVirtual=True).first()
 
-def find_offerer_by_validation_token(token: str) -> Optional[UserOfferer]:
-    return Offerer.query.filter_by(validationToken=token).one_or_none()
 
+def has_venue_missing_bank_information(offerer_id: int) -> bool:
```

Comment:
La formulation me pique un peu. On ne pourrait pas faire quelque chose de plus clair comme (par exemple):
```suggestion
def does_venue_have_empty_bank_information(offerer_id: int) -> bool:
```

Reply:
Il me semble qu'il y a une habitude de nommage des fonctions retournant un bool, à savoir préfixer par `is` ou `has`.
`has_venue_without_bank_information()` peut-être ?

Reply:
Dans ce cas, `is_venue_missing_bank_information` me parait plus anglais 
"I am missing a piece of information" me choque moins que "I have a piece of information missing"
Si c'est missing, je peux pas trop le have

Reply:
sauf que `is_venue_missing_bank_information` porterai sur une venue et non un offerer.
`is_any_venue_missing_bank_information` ?

Reply:
`has_venue_without_bank_information()` est l'heureux élu.